### PR TITLE
Add showcase

### PR DESCRIPTION
**Pull request by isabloCh**
added book list output
update ruby version 2.5.0 => 2.5.1

### CODE REVIEW

File: app/javascript/index_books.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<template>
+  <div id="book">
+    <ul v-for="book in books">
+      <h5> {{book.title}} </h5>
```
Comment (resivalex) [inline app/javascript/index_books.vue]: Don't rely on html element formatting. Prefer css classes instead

```html
<div class="book__title">{{book.title}}</div>
<div class="book__description">{{book.description}}</div>
```

File: app/views/books/index.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<div id = "books_list" data = "<%= @books.to_json %>" />
```
Comment (resivalex) [inline app/views/books/index.html.erb:1]: Remove spaces around `=`

**Review by resivalex — CHANGES_REQUESTED** [review]

File: app/javascript/index_books.vue (hunk 1 above)
Comment (resivalex) [reply app/javascript/index_books.vue]: лучше вынеси прорисовку одной книги в отдельный vue-файл и тогда можно использовать классы `.title` и `.description`. у тебя ведь `scoped`

**Review by resivalex — CHANGES_REQUESTED** [review]

File: app/javascript/index_books.vue [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      <p> {{book.description}} </p>
+    </ul>
+  </div>
+</template>
+
+<script>
+  export default {
+    data: function () {
+      let el = document.getElementById('books_list');
+      let json = JSON.parse(el.getAttribute('data'));
```
Comment (Loremaster) [inline app/javascript/index_books.vue]: I would recommend to use something like [axios](https://github.com/axios/axios) to fetch data over api. It's never a good idea to fetch data from data attributes in vue

File: app/javascript/index_books.vue [hunk 4]
```diff
@@ -0,0 +1,31 @@
+<template>
```
Comment (Loremaster) [inline app/javascript/index_books.vue:1]: The name of the file is not really good. Better one would be `books_list`

File: app/javascript/packs/application_vue.js [hunk 5]
```diff
@@ -6,16 +6,17 @@
...
-    el,
-    render: h => h(App)
-  })
+  let el = document.getElementById('books_list');
+  let indexBooks = new Vue({
+    render: h => h(IndexBooks),
+  }).$mount(el)
 
-  console.log(app)
+  console.log(indexBooks);
```
Comment (Loremaster) [inline app/javascript/packs/application_vue.js]: Remove this logging please

File: app/javascript/packs/application_vue.js [hunk 6]
```diff
@@ -6,16 +6,17 @@
...
+
+  console.log(json);
```
Comment (Loremaster) [inline app/javascript/packs/application_vue.js]: And this one. Also install [vue devtools](https://chrome.google.com/webstore/detail/vuejs-devtools/nhdogjmejiglipccpnnnanhbledajbpd?hl=en)

File: app/views/books/index.html.erb (hunk 2 above)
Comment (Loremaster) [reply app/views/books/index.html.erb:1]: You don't need this line if you will fetch data over API (highly recommended)

**Review by Loremaster — CHANGES_REQUESTED** [review]

File: app/views/books/index.html.erb (hunk 2 above)
Comment (resivalex) [reply app/views/books/index.html.erb:1]: Agree. The problem is the data doesn't fetch from API yet

File: app/javascript/index_books.vue (hunk 3 above)
Comment (isabloCh) [reply app/javascript/index_books.vue]: Do you want my death?

Comment (resivalex) [reply app/javascript/index_books.vue]: Let's force @isabloCh to do it in the next pull-request

File: app/javascript/index_books.vue (hunk 4 above)
Comment (resivalex) [reply app/javascript/index_books.vue:1]: 👍

File: app/javascript/index_books.vue (hunk 3 above)
Comment (Loremaster) [reply app/javascript/index_books.vue]: @resivalex alright 👍

**Comment by Loremaster** [comment]
@isabloCh I will approve your PR after few small changes I requested here. You can implement fetching data from API later

**Comment by Loremaster** [comment]
@isabloCh You should also correct your PR description. You update loofah to `2.2.2`.

File: app/javascript/packs/application_vue.js (hunk 6 above)
Comment (isabloCh) [reply app/javascript/packs/application_vue.js]: long added

**Review by resivalex — APPROVED** [review]

**Pull request closed without merging**